docs/tutorials/notebooks/tracking_progress.ipynb

```diff
@@ -0,0 +1,351 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    from ott.utils import default_progress_fn
</pre><p>For consistency, I'd prefer <code>from ott import utils</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/347/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Sure, that's indeed more consistent.</p>